utils/src/main/java/software/amazon/awssdk/utils/DateUtils.java

```diff
@@ -68,11 +74,22 @@ public static Instant parseIso8601Date(String dateString) {
                              .concat("Z");
```

Comment:
Do we still have to have lines 70-75?

Reply:
Turns out we still need them because `+0000` is not a valid offset; it should be `+00:00`

Reply:
Bleh, okay.